codecov.yml

```diff
@@ -15,16 +8,14 @@ coverage:
       frontend:
         # codecov will not fail status checks for master
         only_pulls: true
-        # This will be needed until codecov supports after_n_builds for flags
-        informational: true # Do not fail the check
+        informational: false # Fail the check
         target: 60%
         flags:
         - frontend
       backend:
         # codecov will not fail status checks for master
         only_pulls: true
-        # This will be needed until codecov supports after_n_builds for flags
-        informational: true # Do not fail the check
+        informational: false # Fail the check
```

Comment:
There would be value in breaking this change into two parts:

1. Enable the comments
2. Set informational to false

That was if there are any issues with the coverage status check, reverting it won't disable the comments.